styles.css

```diff
@@ -0,0 +1,72 @@
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Inter', sans-serif;
+}
+
+body {
+  padding: 24px;
+}
+
+a {
+  color: black;
+  text-decoration: none;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+}
+
+header h2 {
+  font-weight: 600;
+}
+
+header .fa-git {
+  font-weight: 500;
+  font-size: 20px;
+  cursor: pointer;
+}
+
+.home-page {
```

Comment:
- [ ] Great job positioning the sub-contents of this section, but it'd be better if you used Flexbox on the container to position the items, I understand they already look great, but one of the goals of this project is to greatly improve your proficiency with Flexbox, so kindly use Flex on this section to help position the sub-items and adjust the margins accordingly😃
![image](https://user-images.githubusercontent.com/73140461/143514114-fd0934e6-568a-46be-89bd-026862f2cffa.png)